plugins/jobs/web_client/views/timeChartConfig.js

```diff
@@ -270,7 +270,7 @@ export default {
     'legends': [
         {
             'fill': 'timing',
```

Comment:
@matthewma7 This plot still renders with no legend icons. What is the purpose of the legend on this page?